config/example/config.txt

```diff
@@ -474,3 +474,6 @@ ENABLE_AUTO_PROFILER
 
 ## Uncomment to enable map voting
 #ENABLE_MAP_VOTING
+
+## Host for a custom 2FA server. Comment to disable. Do not use a trailing slash or https.
```

Comment:
> Do not use [..] https.

???

Reply:
Originally this was using world.Export() before it became only compatible because yay BYOND.


The actual 2FA backend runs on the local subnet for paradise infrastructure and has no route to the outside world without being proxied through BYOND. 

It also refuses to show an OTP QR code twice for a user unless they deactivate their old one.